cmd/xmidt-agent/config.go

```diff
@@ -244,11 +246,13 @@ func provideConfig(cli *CLI) (*goschtalt.Config, error) {
 				goschtalt.ValidatorFunc(validate.Validate),
 			),
 		),
+		// Seed the program with the default, built-in configuration
+		goschtalt.AddBuffer("build-in.yaml", defaultConfigFile, goschtalt.AsDefault()),
```

Comment:
Maybe use the name `!built-in.yaml` so this is lexigraphically very very early.  See asciitable.com for the ordering.

Reply:
that's so cool, never knew that!